docs/guides/qiskit-addons-cutting-gates.ipynb

```diff
@@ -0,0 +1,470 @@
+      "text/plain": [
+       "<Figure size 1434x294.311 with 1 Axes>"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit.circuit.library import efficient_su2\n",
+    "from qiskit.quantum_info import SparsePauliOp\n",
+    "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
+    "from qiskit_ibm_runtime.fake_provider import FakeManilaV2\n",
+    "from qiskit_ibm_runtime import SamplerV2, Batch\n",
+    "from qiskit_aer.primitives import EstimatorV2\n",
+    "from qiskit_addon_cutting import (\n",
+    "    cut_gates,\n",
+    "    partition_problem,\n",
+    "    generate_cutting_experiments,\n",
+    "    reconstruct_expectation_values,\n",
+    ")\n",
+    "\n",
+    "circuit = efficient_su2(num_qubits=4, entanglement=\"circular\")\n",
+    "circuit.assign_parameters([0.4] * len(circuit.parameters), inplace=True)\n",
+    "\n",
+    "\n",
+    "observable = SparsePauliOp([\"ZZII\", \"IZZI\", \"-IIZZ\", \"XIXI\", \"ZIZZ\", \"IXIX\"])\n",
+    "print(f\"Observable: {observable}\")\n",
+    "circuit.draw(\"mpl\", scale=0.8)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "87b8819b-75d7-4a9e-b91a-00b6155fda9a",
+   "metadata": {},
+   "source": [
+    "Each of the [`CNOT`](/api/qiskit/qiskit.circuit.library.CXGate) gates between qubits $q_0$ and $q_3$ introduce two swap gates after transpilation (assuming the qubits are connected in a straight line). To avoid this increase in depth, you can replace these distant gates with [`TwoQubitQPDGate`s](/api/qiskit-addon-cutting/qpd-two-qubit-qpd-gate) using the [`cut_gates()`](/api/qiskit-addon-cutting/qiskit-addon-cutting#cut_gates) method.  This function also returns a list of [`QPDBasis`](../api/qiskit-addon-cutting/qpd-qpd-basis) instances - one for each decomposition."
```

Comment:
```suggestion
    "Each of the [`CNOT`](/api/qiskit/qiskit.circuit.library.CXGate) gates between qubits $q_0$ and $q_3$ introduce two SWAP gates after transpilation (assuming the qubits are connected in a straight line). To avoid this increase in depth, you can replace these distant gates with [`TwoQubitQPDGate`s](/api/qiskit-addon-cutting/qpd-two-qubit-qpd-gate) using the [`cut_gates()`](/api/qiskit-addon-cutting/qiskit-addon-cutting#cut_gates) method.  This function also returns a list of [`QPDBasis`](../api/qiskit-addon-cutting/qpd-qpd-basis) instances - one for each decomposition."
```